Tests/TestingApps/SimpleApp/SimpleApp/Views/AppContentView.swift

```diff
@@ -30,60 +31,86 @@ struct AppContentView: View {
     @State
     private var customerInfo: CustomerInfo?
 
+    @State
+    private var didPurchase: Bool = false
+
     var body: some View {
         ZStack {
-            Rectangle()
```

Comment:
This is mostly indentation changes from moving it to a separate property. I recommend viewing this without whitespaces.